back/app/services/project_copy_service.rb

```diff
@@ -166,6 +166,9 @@ def yml_custom_fields(shift_timestamps: 0)
         'maximum' => field.maximum,
         'minimum_label_multiloc' => field.minimum_label_multiloc,
         'maximum_label_multiloc' => field.maximum_label_multiloc,
+        'select_count_enabled' => field.select_count_enabled,
```

Comment:
Good that you thought about the project copy, but we also need to include it in the tenant template serializer.